hedy.py

```diff
@@ -1516,12 +1520,17 @@ def assign(self, args):
         else:
             # we no longer escape quotes here because they are now needed
             return parameter + " = " + value + ""
-    
+
```

Comment:
I am not sure why we need to add this? We already have such a var() in level 2: https://github.com/Felienne/hedy/blob/305f949991a6c344e42c2231224eeb994e412317/hedy.py#L1102

Reply:
Ah, I see, the answer seems to be: "because if we don't add this, the tests stop working"

But it should work I am diving into this.